doc/install/monasca-log-api.md

```diff
@@ -1,45 +1,47 @@
 ## Configuring the Monasca Log API
-The Monasca Log API runs on top of a [gunicorn][1] server. For a minimal configuration, a few settings have to be tuned in the following files:
-
-* [docker-compose.yml][4]
-* [monasca-log-api/Dockerfile][5]
-* [monasca-log-api/log-api-config.ini.j2][6]
-* [monasca-log-api/log-api-config.conf.j2][7]
-
-#### Compose file
-In _docker-compose.yml_, look up the service named 'monasca-log-api'. Here you might want to edit the `port` (setting is `traefik.port`) the service listens to (default is `8090`: change this value carefully, updating it also in the `server:main` section of _log-api-config.ini.j2_ and in the `output` section of _logstash.conf_ on the [Monasca Log Agent](monasca-log-agent.md)). Also, the `restart` policy of the service is set to `on-failure` to guarantee the service stays up. You might want to [edit][2] this value or even remove this setting.
-
-#### Dockerfile & Enviroment Variables
-There should be no need to change default settings, as the Docker image ([martel/monasca-log-api][10]) provides good defaults.
-
-However, settings can be overridden by changing values of [environment variables](https://docs.docker.com/compose/compose-file/#environment) in the compose file (see the `environment` section of services in [docker-compose.yml][4] and [docker-compose-keystone.yml][9]).
-
-#### Other configuration files
-In *log-api-config.ini*, the main settings are the `host` IP address and `port` the gunicorn server should listen to, as well as the number of working processes (`workers`) for handling requests, the maximum number of simultaneous clients (`worker-connections`), and the maximum number of pending connections (`backlog`). Please refer to the [settings][3] section in [gunicorn's documentation][1].
-
-In *log-api-config.conf*, the relevant settings are: the `region` (in the `service` section), that will be added to the logs as metadata; in the `log_publisher` section, the Kafka `topics` to which the logs are published and the address of the Kafka broker (`kafka_url`).
+The Monasca Log API (docker image [here](https://hub.docker.com/r/martel/monasca-log-api/)) runs on top of a [gunicorn](http://docs.gunicorn.org/en/latest/) server. Settings can be tuned via environment variables and deployment options in the [compose file](https://github.com/martel-innovate/deep-log-inspection/blob/master/log-server/docker-compose.yml) (look up the service named 'monasca-log-api').
```

Comment:
link to docker image can be updated an probably the image martel/monasca-log-api deleted to make there's no remaining reference to it anywhere else.